libdnf/utils/os-release.cpp

```diff
@@ -150,4 +150,13 @@ std::string getUserAgent(const std::map<std::string, std::string> & osReleaseDat
     return oss.str();
 }
 
+std::string getUserAgent()
+{
+    std::map<std::string, std::string> osdata;
+    try {
+        osdata = getOsReleaseData();
+    } catch (...) {}
```

Comment:
Just browsing through: I think it's bad to silently ignore any exceptions, even if you don't need the code in the `try` block to succeed.

This may start failing on an arbitrary issue and (presumably, in this case) not send the countme data (or not in the correct form). It may take someone considerable amount of time figuring out why, including the need to come here and add like a log warning, wait for a release or build themself, and then go and find out what's going on.

The proper thing to do here seems to log a warning with the text of the caught exception.